src/openforms/forms/api/serializers/logic/action_serializers.py

```diff
@@ -72,6 +73,17 @@ class LogicFetchActionSerializer(serializers.Serializer):
     )
 
 
+class VariableMappingSerializer(serializers.Serializer):
+    form_variable = serializers.CharField(
+        required=True,
+        validators=[variable_key_validator],
+        help_text=_("Key of the form variable."),
```

Comment:
```suggestion
        label=_("Key of the form variable."),
```
help text is more intended for longer descriptive texts. There should always be a label, but a help text without label looks off :)